### ISSUE TITLE
Umbrel Integration Does Not Fire Off Timed Autofee Scripts

### ISSUE THREAD

**Issue by Xian001** [cryptosharks131/lndg#65]
None of the auto-fee scripts are being fired with the Umbrel integration of lndg.

I need to manually wrangle the fee's page at least a couple times a day.

Would be nice to have options for it to run/adjust fees at 3 / 6 / 12 and 24 hour intervals (or arbitrary...)

**Comment by cryptosharks131** [comment]
The version currently available on Umbrel (v1.0.5) does have an issue that causes the AF not to trigger properly for some users.

Version v1.1.1 should resolve this issue which is currently in a PR to be added to the next Umbrel update.

If you are interested in updating to v1.1.1 early you can use the following steps.
```
Stop LNDg
~/umbrel/scripts/app stop lndg

Update docker-compose line #5
nano ~/umbrel/apps/lndg/docker-compose.yml

Update line 5 after image to this:
ghcr.io/cryptosharks131/lndg:v1.1.1@sha256:fd975e4d6ee2de6da932eb8e93648b13ad7de9b71ec2a35eac19d2845d7a5f31

Restart LNDg
~/umbrel/scripts/app start lndg
```

**Comment by Xian001** [comment]
Hey, kickass ! TYVM! Updated and going to see how she works.

Thanks for all your great work!

"LNDg v1.1.1"
